internal/validate/validate.go

```diff
@@ -393,6 +412,43 @@ func CheckConnectorTokens(parameter interface{}, variableName string) error {
 	return nil
 }
 
+func CheckAddressesList(parameter interface{}, variableName string) error {
```

Comment:
For consistency, I would name this `CheckEthereumAddressesList`

Reply:
I will remove it, since it is not used